packages/manager/apps/dedicated/client/app/components/dedicated-cloud/users/vsphere-users/enable/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,7 @@
+{
+  "dedicatedCloud_USER_enable_title": "Activer un utilisateur",
+  "dedicatedCloud_USER_enable_infos": "Cet utilisateur est un administrateur temporaire de votre infrastructure. Il sera désactivé dans 2h.",
+  "dedicatedCloud_USER_enable_message": "Etes-vous certain(e) de vouloir activer l'utilisateur <strong>{{t0}}</strong> ?",
```

Comment:
Suggestions: remove html from TRANS files

Reply:
It's applied.